source/CommonJobs/CommonJobs.Migrations/201212010811_CreateDefaultMenus.cs

```diff
@@ -21,7 +21,8 @@ public override void Up()
         {
             var filePath = Path.Combine(Path.GetDirectoryName(Path.GetDirectoryName(new Uri(GetType().Assembly.CodeBase).LocalPath)), "Import201212010811.json");
             var serializer = new Newtonsoft.Json.JsonSerializer();
-            var placeByUserName = serializer.Deserialize <Dictionary<string, string>>(new StreamReader(filePath));
+            //TODO: VALIDATE THIS
```

Comment:
I didn't test migrations (not sure how). But I changed the StreamReader, so it was kind of a reminder.